### ISSUE TITLE
Missing Dependencies on RHx86_64

### ISSUE THREAD

**Issue by GarretL** [MrMEEE/awx-build#218]
When installing on RH7.7, the ansible-awx package is skipped due to missing dependencies.

Error excerpt
"Requires: rh-python36-build
Requires: xmlsecl-openssl-devel
Requires: xmlsecl-devel"

To carry out the installation, the following two commands should be run before installation:

yum install -y http://mirror.centos.org/centos/7/sclo/x86_64/rh/rh-python36/rh-python36-build-2.0-1.el7.x86_64.rpm
subscription-manager repos --enable rhel-7-server-optional-rpms \
  --enable rhel-server-rhscl-7-rpms

Then it installs fine. Thanks for the guide!

**Comment by MrMEEE** [comment]
Hi

Thanks for the feedback..

I believe this is already covered by part2 in the repository part of the installation guide??

https://awx.wiki/installation/repositories/rhel7-x86_64

**Comment by GarretL** [comment]
So I did use v1 of the install guide due to an ssl error that I saw on my VM: https://imgur.com/a/iqImjsc That's on firefox on RHEL7.7. Chrome on Windows 10 works without any issue for the SSL cert.

When I got  to work, I restored my VM to a fresh checkpoint and redid the installation using the install guide from awx.wiki. I would like to retract the error regarding "rh-python36-build" but the other two still stand.

When I follow the instructions ansible-awx doesn't install due to 
"...Requires: xmlsecl-openssl-devel...
...Requires: xmlsecl-devel..."

To fix this, the following should be executed before installing ansible-awx:
"subscription-manager repos --enable rhel-7-server-optional-rpms"

**Comment by MrMEEE** [comment]
Project has been updated and moved...

Please move to the new version and report any issues

PROJECT MOVED: https://github.com/miracle-as/AWX-RPM

Issues at: https://github.com/miracle-as/AWX-RPM/issues

**Install guide, tools, utilities are located at: https://awx.wiki

**LinkedIn group for Questions, support, talk and more: https://www.linkedin.com/groups/13694893/**
